client/src/components/account/CookiesModal.tsx

```diff
@@ -10,7 +10,7 @@ export default function CookieModal() {
     let location = useLocation()
 
     const handleWrong = () => {
-        setErrMsg("In order to use RapidTyper you need to accept nessessary cookies.")
+        setErrMsg("We are sorry, but in order to use RapidTyper you need to accept nessessary cookies.")
```

Comment:
That might be more user-friendly and not too offensive (I was a bit annoyed by all the red text 😉). However, I think removing it altogether would be a good option, see also #3 .